lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -30,7 +31,7 @@ export class EvaluationClientWrapper {
       restrictToDIDMethods?: string[];
     }
   ): SelectResults {
-    let selectResults: SelectResults;
+    let selectResults: Partial<SelectResults> = {};
```

Comment:
Why does this need to become a partial. Later the selectResults is fully being populated

The other branch is not ding anything with the var and then later the selectResults are casted to SelectResults without modifications in between

Reply:
removed the `Partial`. it was a intermediate step and we no longer needed this